pennylane/devices/default_qubit.py

```diff
@@ -843,20 +853,23 @@ def _simulate_wrapper(circuit, kwargs):
 
 
 def _adjoint_jac_wrapper(c, debugger=None):
+    c = c.map_to_standard_wires()
```

Comment:
We had been calling it multiple times inside of the simulation for some reason.  Now we aren't calling it multiple times throughout the simulation, but just once at the start of the simulation.

I would consider this more intuitive than calling it over and over again throughout all the various simulation utilities.

It needs to be called, so where is called? Once before we start any simulation code.

Reply:
Alright, that is fair!

Reply:
I rely on bypassing these calls in #5180 . There may be a better way, but is there a way to undo it if needed?

Reply:
Mind providing some more context for why it doesn't work at a higher point in the stack?

Would it you help if we instead mimicked lightning and mapped according to a pre-initialized `_wire_map` instead of using a more dynamic wire mapping?